nixos/modules/services/web-servers/apache-httpd/wordpress.nix

```diff
@@ -137,9 +137,33 @@ in
     };
     dbPassword = mkOption {
       default = "wordpress";
-      description = "The mysql password to the respective dbUser.";
+      description = ''
+        The mysql password to the respective dbUser.
+
+        Warning: this password is stored in the world-readable Nix store. It's
+        recommended to use the $dbPasswordFile option since that gives you control over
+        the security of the password. $dbPasswordFile also takes precedence over $dbPassword.
+      '';
       example = "wordpress";
     };
+    dbPasswordFile = mkOption {
+      type = types.str;
+      default = toString (pkgs.writeTextFile {
```

Comment:
@globin note that wordpress is not mentioned in the manual.

@LnL7 `wordpress.nix` is not a traditional NixOS module and doesn't have a `config` attribute. It should be used as:

```nix
{ 
  services.httpd.extraSubservices = [
    { serviceType = "wordpress";
      dbName = "...";
      themes = [];
      plugins = [];
    }
  ];
}
```

So there's no need and no place to use `mkDefault`.